realm/realm-library/src/main/java/io/realm/DynamicRealmObject.java

```diff
@@ -1252,6 +1284,10 @@ public String toString() {
                 case UUID:
                     sb.append(proxyState.getRow$realm().isNull(columnKey) ? "null" : proxyState.getRow$realm().getUUID(columnKey));
                     break;
+                case MIXED:
+                    //TODO: MIXED VALIDATE STRING CONVERSION
```

Comment:
Is this anything wrong with this?

Reply:
It will be addressed when working on bulk inserts on another PR.